xsuite/src/world/signer.ts

```diff
@@ -97,6 +121,10 @@ export class Keystore {
     return UserWallet.decryptMnemonic(this.data, this.password).getWords();
   }
 
+  getPrivateKey() {
+    return UserWallet.decryptSecretKey(this.data, this.password).hex();
```

Comment:
Here I return `.hex()` but maybe it could also be a `Buffer` using `.valueOf()` or even a `string` if we apply `.toString()` to this `Buffer`.

Reply:
After implementing #71 , I figured the best option was removing `.hex()` here.